README.md

```diff
@@ -22,6 +22,44 @@ An alternative command is:
 `java -cp ".:imagelab.jar" Run`  
 Note that on Windows platforms, the ":" character in the classpath must be changed to the ";" character.
 
+## Build Instructions 
```

Comment:
I think the reader is still left without a clear answer to the  question, "How do I build the product?"

Why not prominently feature the command
```./gradlew build```

The information articulated in this section is very useful and should be kept.
My concern is with the user experience.